editorextensions/codebeamer-custom-ui/src/importmodal.ts

```diff
@@ -89,9 +142,10 @@ export class ImportModal extends Modal {
 			if (cardData.style)
 				block.properties.set('LineColor', cardData.style.cardTheme);
 			block.setDescription(' '); // add empty description to disable 'Description' placeholder on created cards
+			block.shapeData.set('codebeamerItemId', cardData.codebeamerItemId);
 		}
 
-		console.log("Card created: ", cardData.title);
+		console.log('Card created: ', cardData.title);
```

Comment:
Let's stop logging every single card and potentially massively spamming the console with every import.